packages/blockchain-wallet-v4-frontend/src/components/Flyout/index.tsx

```diff
@@ -96,6 +96,38 @@ export const FlyoutChild = styled.div`
   height: 100%;
   width: 100%;
 `
+export const Row = styled.div<{
+  color?: string
+}>`
+  padding: 16px 40px;
+  box-sizing: border-box;
+  border-top: 1px solid
+    ${props => (props.color ? props.theme[props.color] : props.theme.grey800)};
+  &:last-child {
+    border-bottom: 1px solid
+      ${props => (props.color ? props.theme[props.color] : props.theme.grey800)};
+  }
+`
+export const Title = styled(Text)<{
+  color: string
+  size?: string
+  weight?: number
+}>`
+  font-size: ${props => props.size};
+  font-weight: ${props => props.weight};
+  color: ${props => props.theme[props.color]};
+`
+export const Value = styled(Text)<{
+  color: string
+  size?: string
+  weight?: number
+}>`
```

Comment:
Pretty sure all Values are the same, we don't need to set color, size, weight in the components, just set them here